rust/crd/src/lib.rs

```diff
@@ -106,12 +107,10 @@ pub struct AirflowClusterSpec {
     /// Emergency stop button, if `true` then all pods are stopped without affecting configuration (as setting `replicas` to `0` would)
     #[serde(default, skip_serializing_if = "Option::is_none")]
     pub stopped: Option<bool>,
-    #[serde(default, skip_serializing_if = "Option::is_none")]
-    pub version: Option<String>,
+    /// The Airflow image version to use
```

Comment:
Nit: By using custom it can be completely other image
```suggestion
    /// The Airflow image to use
```